### ISSUE TITLE
Description field on resources not parsing/ignoring certain unicode symbols

### ISSUE THREAD

**Issue by c-ameron** [cloudflare/terraform-provider-cloudflare#771]
Hi, it looks like the API is storing the description for resources in unicode, and its not being properly translated (or ignored) in Terraform. Could we maybe ignore certain symbols on a diff? Thanks!

### Terraform Version
Terraform v0.12.28
provider.cloudflare v2.9.0

### Affected Resource(s)
- cloudflare_firewall_rule
- cloudflare_filter

### Terraform Configuration Files
```hcl
resource "cloudflare_filter" "botnet_le_10" {
  zone_id    = "yy"
  description = "Botnet Score < 10"
  expression = "(cf.bot_management.score lt 10)"
}

resource "cloudflare_firewall_rule" "botnet_lt_10" {
  zone_id     = "yy"
  description = "Botnet Score < 10"
  filter_id   = cloudflare_filter.botnet_le_10.id
  action      = "log"
}
```

### Expected Behavior
No change after creation

### Actual Behavior
Upon successful creation, apply again and you'll always get the diff

```hcl
  # cloudflare_firewall_rule.botnet_lt_10 will be updated in-place
~ resource "cloudflare_firewall_rule" "botnet_lt_10" {
        action      = "log"
      ~ description = "Botnet Score &lt; 10" -> "Botnet Score < 10"
        filter_id   = "123"
        id          = "xx"
        paused      = false
        products    = []
        zone_id     = "yy"
    }

  # cloudflare_filter.botnet_lt_10 will be updated in-place
~ resource "cloudflare_filter" "botnet_lt_10" {
      ~ description =  "Botnet Score &lt; 10" -> "Botnet Score < 10"
        expression  = "(cf.bot_management.score lt 10)"
        id          = "xx"
        paused      = false
        zone_id     = "yy"
    }
```

### Steps to Reproduce
1. `terraform apply` 

### Important Factoids
I'm seeing from the API a similar issue with symbols being stored as `"description": "Botnet Score \u0026lt; 10"`

I know I can ignore it with an `ignore_changes` block but thought I'd create a ticket anyway for visibility. Thanks!!

**Comment by jacobbednarz** [comment]
This was fixed in #758 and is pending a release. You're welcome to use master for now if you need it.

**Comment by c-ameron** [comment]
Ah awesome! great timing, thanks!! :)
